Swat/SwatTableView.php

```diff
@@ -474,8 +494,13 @@ public function hasMessage()
 	public function getXhtmlColspan()
 	{
 		$count = 0;
-		foreach ($this->getVisibleColumns() as $column)
+		foreach ($this->getVisibleColumns() as $column) {
 			$count += $column->getXhtmlColspan();
+		}
+
+		if ($count < 2 && count($this->getSpanningColumns()) > 0) {
```

Comment:
The title for a spanning column is optional. If it is not set only one column is used.